src/jama/create/create.py

```diff
@@ -46,10 +49,17 @@ def from_dialog(url_base, json_to_parse):
         to_post_obj["itemType"] = int(sub_data["itemType"])
         to_post_obj["fields"]["name"] = sub_data["newItemName"]
         to_post_obj["fields"]["description"] = sub_data["description"]
-        to_post_obj["fields"]["assignedTo"] = sub_data["asignee"]
 
-        jama_resp = requests.post(jama_url, json=to_post_obj, auth=(os.environ['JAMA_USER'], os.environ['JAMA_PASS']))
-            
+        if "DB_HOST" not in os.environ:
```

Comment:
You can simply use the `api_caller.post()` in `jama/api_caller` to do the same thing.
I think using the same caller function can make our code more maintainable.